.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,29 @@
 ## PR Checklist
 Please check if your PR fulfills the following requirements:
 
-- [ ] The commit message follows our guidelines: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide
+- [ ] Tests for the changes have been added (for bug fixes / features)
```

Comment:
I think this should be change to something like `Changes have been rendered and validate locally using mkdocs-material`

This is the only "testing" we can do for docs and only applies to changes to docs_src